app/services/group_service.rb

```diff
@@ -1,6 +1,6 @@
 module GroupService
-  def self.ab_test_alternative?(group)
-    group.segment_seed % 2 == 0
+  def self.ab_testing_enabled?
+    Rails.env.production? or ENV['LOOMIO_AB_TESTING_ENABLED']
```

Comment:
Yea.. walking home tonight, I suddendly realised that the production condition was wrong.. I thought to myself.. this is why we have code review!